stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -101,6 +102,16 @@ class Value():
     def __init__(self, typecode_or_type: str, *args: Any, lock: bool = ...) -> None: ...
 
 # ----- multiprocessing function stubs -----
+def active_children() -> List[Process]: ...
+def allow_connection_pickling() -> None: ...
 def cpu_count() -> int: ...
 def freeze_support() -> None: ...
+def get_all_start_methods() -> List[str]: ...
+def get_context(method: Optional[str] = ...) -> BaseContext: ...
+def get_logger() -> Logger: ...
+def get_start_method(allow_none: Optional[bool] = ...) -> str: ...
+def log_to_stderr(level: Optional[str] = ...) -> Logger: ...
```

Comment:
I think the level can also be an int, since it just gets passed to setLevel in logging.